src/end.sol

```diff
@@ -4,7 +4,6 @@
 
 // Copyright (C) 2018 Rain <rainbreak@riseup.net>
 // Copyright (C) 2018 Lev Livnev <lev@liv.nev.org.uk>
-// Copyright (C) 2020-2021 Maker Ecosystem Growth Holdings, INC.
```

Comment:
Please add this line back with the Dai Foundation. We introduced changes to the end during those years, that's why the copyright was added.

Reply:
done